### DISCUSSION TITLE
Bridging to Denon's HEOS speakers?

### DISCUSSION THREAD

**Discussion by kristianrl** [philippe44/AirConnect#399] [Q&A] [answered]
I have a couple of [Denon HEOS 1 HS2](https://www.denon.com/en-au/shop/speakers/heos1hs2)-speakers. I currently connect to them using Bluetooth (they are paired and act as a single stereo Bluetooth speaker).

Is it by any means possible to use AirConnect as a bridge for these HEOS 1-speakers?

Denon provides documentation on their [HEOS telnet interface](https://www.denon.com/-/media/files/documentmaster/denonna/heos_cli_protocolspecification-version.pdf).

**Comment by philippe44** [comment, marked as answer]
It's a proprietary protocol so that would be a full implementation which is way over my available time.

**Comment by kristianrl** [comment]
[Denon HEOS 1 HS2](https://www.denon.com/en-au/shop/speakers/heos1hs2)-speakers turn out to be supported.
